tests/firstAutotests.spec.ts

```diff
@@ -0,0 +1,82 @@
+import { test, expect } from '@playwright/test';
+
+test('Verify login with valid credentials', async ({ page }) => {
+  await page.goto('https://practicesoftwaretesting.com/auth/login');
+
+  await page.getByPlaceholder('Your email').fill('customer@practicesoftwaretesting.com');
+  await page.getByPlaceholder('Your password').fill('welcome01');
+
+  await page.click('.btnSubmit');
+
+  await expect(page).toHaveURL('https://practicesoftwaretesting.com/account');
+
+  const MyAcc = page.locator('[data-test="page-title"]');
```

Comment:
I'd use the getByTestId method to find an element. But you must do this before https://playwright.dev/docs/api/class-testoptions#test-options-test-id-attribute